src/main/resources/application-prod.yml

```
@@ -53,4 +53,9 @@ server:
 
 urls:
   frontend: http://localhost:3000
-  backend: http://43.202.178.60:8080
\ No newline at end of file
+  backend: http://43.202.178.60:8080
+
+google:
+  client-id: ${GOOGLE_CLIENT_ID}
+  client-secret: ${GOOGLE_CLIENT_SECRET}
+  redirect-uri: https://developers.google.com/oauthplayground
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)